sample_files/pre-commit-13.0/.eslintrc

```
@@ -0,0 +1,289 @@
+    "no-restricted-imports": "warn",
+    "no-restricted-modules": "warn",
+    "no-restricted-syntax": "warn",
+    "no-return-assign": "warn",
+    "no-script-url": "warn",
+    "no-self-assign": "warn",
+    "no-self-compare": "warn",
+    "no-sequences": "warn",
+    "no-shadow": "warn",
+    "no-shadow-restricted-names": "warn",
+    "no-whitespace-before-property": "warn",
+    "no-spaced-func": "warn",
+    "no-sparse-arrays": "warn",
+    "no-sync": "warn",
+    "no-tabs": "warn",
+    "no-ternary": "off",
+    "no-trailing-spaces": "warn",
+    "no-this-before-super": "warn",
+    "no-throw-literal": "warn",
+    "no-undef": "warn",
+    "no-undef-init": "warn",
+    "no-undefined": "off",
+    "no-unexpected-multiline": "warn",
+    "no-underscore-dangle": "off",
+    "no-unmodified-loop-condition": "warn",
+    "no-unneeded-ternary": "warn",
+    "no-unreachable": "warn",
+    "no-unsafe-finally": "warn",
+    "no-unused-expressions": "warn",
+    "no-unused-labels": "warn",
+    "no-unused-vars": "warn",
+    "no-use-before-define": "warn",
+    "no-useless-call": "warn",
+    "no-useless-computed-key": "warn",
+    "no-useless-concat": "warn",
+    "no-useless-constructor": "warn",
+    "no-useless-escape": "warn",
+    "no-useless-rename": "warn",
+    "no-void": "warn",
+    "no-var": "off",
+    "no-warning-comments": "off",
+    "no-with": "warn",
+    "array-bracket-spacing": "off",
+    "array-callback-return": "warn",
+    "arrow-body-style": "warn",
+    "arrow-parens": "warn",
+    "arrow-spacing": "off",
+    "accessor-pairs": "warn",
+    "block-scoped-var": "off",
+    "block-spacing": ["warn", "always"],
+    "brace-style": "warn",
+    "callback-return": "warn",
+    "camelcase": "off",
+    "capitalized-comments": ["warn", "always", {
+      "ignoreConsecutiveComments": true,
+      "ignoreInlineComments": true
+    }],
+    "comma-dangle": ["warn", "always-multiline"],
+    "comma-spacing": ["warn", {
+      "before": false,
+      "after": true
+    }],
+    "comma-style": "warn",
+    "complexity": [
+      "warn",
+      15
+    ],
+    "computed-property-spacing": "off",
+    "consistent-return": "off",
+    "consistent-this": "off",
+    "constructor-super": "warn",
+    "curly": "warn",
+    "default-case": "off",
+    "dot-location": ["warn", "property"],
+    "dot-notation": "warn",
+    "eol-last": "warn",
+    "eqeqeq": "warn",
+    "func-names": "off",
+    "func-style": "off",
+    "generator-star-spacing": "off",
+    "global-require": "warn",
+    "guard-for-in": "off",
+    "handle-callback-err": "warn",
+    "id-blacklist": "warn",
+    "id-length": "off",
+    "id-match": "warn",
+    "indent": "warn",
+    "init-declarations": "warn",
+    "jsx-quotes": "warn",
+    "key-spacing": "off",
+    "keyword-spacing": "warn",
+    "linebreak-style": [
+      "warn",
+      "unix"
+    ],
+    "lines-around-comment": "warn",
+    "max-depth": "warn",
+    "max-len": ["warn", {
+      "code": 80,
```

Let's match the new standard.

```suggestion
      "code": 88,
```

---

80 + bugbear B950 is the recommended black config, so it should be ok (in my experience it works nicely like this). https://github.com/psf/black#line-length

---

I confirm, 80 is the correct value since it's 80 + 10% -> 88.

---

Sorry I don't understand. :confused:  This is for ESLint, not for Python, and we chose 80 to be consistent with Python, so if our new standard for Python is gonna be 88, then it should be for JS too.

---

Ah, sorry I did not notice we where in eslint here. Fine with me then.